webvtt/rendering/cues-with-video/processing-model/support/bidi_vertical_lr.vtt

```diff
@@ -0,0 +1,4 @@
+WEBVTT
+
+00:00:00.000 --> 00:00:05.000 vertical:lr
+<ruby>左<rt>ひだり</rt></ruby>に<ruby> 見<rt>み</rt></ruby>えるのは...
```

Comment:
Shouldn't this test produce more than one line so that the direction of layout of lines can be verified?

Reply:
To distinguish left to right from right to left, you do want a second line, you're right.